### ISSUE TITLE
Error when project in Kentico Cloud has not activated the delivery API

### ISSUE THREAD

**Issue by hades200082** [Kentico/cloud-generators-net#44]
The Delivery API is not active by default on new projects.

This causes an unhandled exception when trying to run the generator against that project.

```
Unhandled Exception: System.InvalidOperationException: Sequence contains no elements
   at System.Linq.Enumerable.Aggregate[TSource](IEnumerable`1 source, Func`3 func)
   at CloudModelGenerator.TypeProviderCodeGenerator.GenerateCode() in C:\projects\cloud-generators-net\src\CloudModelGenerator\TypeProviderCodeGenerator.cs:line 46
   at CloudModelGenerator.CodeGenerator.GenerateTypeProvider() in C:\projects\cloud-generators-net\src\CloudModelGenerator\CodeGenerator.cs:line 61
   at CloudModelGenerator.Program.<>c__DisplayClass4_0.<Main>b__0() in C:\projects\cloud-generators-net\src\CloudModelGenerator\Program.cs:line 57
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at CloudModelGenerator.Program.Main(String[] args) in C:\projects\cloud-generators-net\src\CloudModelGenerator\Program.cs:line 67
```

**Comment by petrsvihlik** [comment]
Right, the app should output a friendly message.

**Comment by kashifsoofi** [comment]
Is there a project id to be used for testing?

**Comment by petrsvihlik** [comment]
You can register at https://app.kenticocloud.com/ for free and use the generated sample project or you can create a new project and make sure you don't enable the Delivery API.

Also, it shouldn't throw an exception when you use a random Guid (Guid that's not associated with any project). I guess the API will treat both situations in the same way so this could be another way to test it.

**Comment by kashifsoofi** [comment]
This can be closed if the solution implemented is acceptable.

**Comment by petrsvihlik** [comment]
it's great that now it doesn't throw an exception. 
i think we should go a one step further and show a friendly message (see above) instead of:
>0 content type models were successfully created.
>CustomTypeProvider class was not created.

because this is quite confusing

**Comment by JanLenoch** [comment]
Fixed by #49 .
